ddtrace/tracer/textmap.go

```diff
@@ -165,13 +169,19 @@ func getPropagators(cfg *PropagatorConfig, env string) []Propagator {
 	if cfg.B3 {
 		defaultPs = append(defaultPs, &propagatorB3{})
 	}
+	if cfg.SQLCommentInjectionMode > CommentInjectionDisabled {
```

Comment:
This approach assumes a specific order of definition of those constants, or certain values. I'd rather use `!=` to avoid surprises. The outcome should be the same.

Same thing below on line 182.